test.js

```diff
@@ -0,0 +1,338 @@
+'use strict'
+
+const { test } = require('tap')
+const Fastify = require('fastify')
+const basicAuth = require('./index')
```

Comment:
Can you add a test where you demo this with https://github.com/fastify/fastify-auth?